.symfony.bundle.yaml

```diff
@@ -1,3 +1,3 @@
-branches: ["1.x", "1.23.x"]
-maintained_branches: ["1.x", "1.23.x"]
+branches: ["2.x", "1.x"]
+maintained_branches: ["2.x", "1.x"]
```

Comment:
Do you think we need to keep `1.23.x`? I don't event remember why we decided to keep it :sweat_smile: (maybe a BC break in 1.24.... :roll_eyes: )

Reply:
No, let's drop now!